app/components/interview_preferences_review_component.rb

```diff
@@ -23,9 +23,11 @@ def show_missing_banner?
   attr_reader :application_form
 
   def interview_preferences_form_row
+    preferences = @interview_preferences_form.interview_preferences.presence || t('application_form.personal_statement.interview_preferences.no_value')
```

Comment:
What's `.presence` here? 🤔

Reply:
Made the change as rubocop told me off for doing:

```ruby
preferences = @interview_preferences_form.interview_preferences.blank? ? t('application_form.personal_statement.interview_preferences.no_value') : @interview_preferences_form.interview_preferences
```

And suggested to do that instead.

So:

```ruby
''.presence
=> nil
'ship it'.presence
=> 'ship it'
''.presence || 'none'
=> 'none'
```

Reply:
Ok 👍